back-end/handlers/playerHandler.ts

```diff
@@ -13,15 +15,15 @@ export default (io: Server, socket: Socket) => {
       const allPlayersInGame = await playerDb.getPlayers(gameId);
       const foundPlayer = allPlayersInGame.find(p => p.name === name);
       const playerWithNameAlreadyExists = !!foundPlayer;
-      const allActiveGameIds = await hostDb.getAllGameIds();
-      const foundGame = allActiveGameIds.find(g => g === gameId);
-      const gameExists = !!foundGame;
+      const allGames: IGame[] = await Game.find({});
+      const foundGames = allGames.find(g => g.id === gameId);
```

Comment:
might want to rename this variable to `foundGame` so it's clear that it's only one game